.github/prepare-nightly_pkg-name.py

```diff
@@ -1,12 +0,0 @@
-import os
```

Comment:
not sure why the package was renamed even this script was never called, so rather remove it completely
https://github.com/PyTorchLightning/pytorch-lightning/runs/1678482450?check_suite_focus=true#step:6:1523